src/main/java/de/tum/in/www1/artemis/config/lti/CustomLti13Configurer.java

```diff
@@ -72,7 +72,8 @@ public void configure(HttpSecurity http) {
         // https://www.imsglobal.org/spec/security/v1p0/#step-3-authentication-response
         OAuth2LoginAuthenticationFilter defaultLoginFilter = configureLoginFilter(clientRegistrationRepository(http), oidcLaunchFlowAuthenticationProvider,
                 authorizationRequestRepository);
-        http.addFilterAfter(new Lti13LaunchFilter(defaultLoginFilter, LTI13_BASE_PATH + LOGIN_PATH, lti13Service(http)), AbstractPreAuthenticatedProcessingFilter.class);
+        String loginPath = "/" + LTI13_BASE_PATH + LOGIN_PATH;
+        http.addFilterAfter(new Lti13LaunchFilter(defaultLoginFilter, loginPath, lti13Service(http)), AbstractPreAuthenticatedProcessingFilter.class);
```

Comment:
Do we really need the slash here? If not, we can remove it. Even if we do I would use the LOGIN_PATH constant (and append it to the slash).
```suggestion
        http.addFilterAfter(new Lti13LaunchFilter(defaultLoginFilter, "/" + LTI13_LOGIN_PATH, lti13Service(http)), AbstractPreAuthenticatedProcessingFilter.class);
```